### PR TITLE
removed unused code in snapshotDao

### PR DESCRIPTION
**Pull request by DaanHoogland**
## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

## Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
Testing
- [ ] I have added tests to cover my changes.
- [ ] All relevant new and existing integration tests have passed.
- [ ] A full integration testsuite with all test that can run on my environment has passed.

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

File: engine/schema/src/main/java/com/cloud/storage/dao/SnapshotDaoImpl.java [hunk 1]
```diff
@@ -214,26 +183,6 @@ protected void init() {
         InstanceIdSearch.done();
     }
 
-    @Override
-    public Long getSecHostId(long volumeId) {
-
-        TransactionLegacy txn = TransactionLegacy.currentTxn();
-        PreparedStatement pstmt = null;
-        String sql = GET_SECHOST_ID;
```
Comment (rafaelweingartner) [inline engine/schema/src/main/java/com/cloud/storage/dao/SnapshotDaoImpl.java]: The `GET_SECHOST_ID` variable is also only used here. So, what about if you delete it as well?

File: engine/schema/src/main/java/com/cloud/storage/dao/SnapshotDaoImpl.java [hunk 2]
```diff
@@ -253,41 +202,6 @@ public long getLastSnapshot(long volumeId, DataStoreRole role) {
         return 0;
     }
 
-    @Override
-    public long updateSnapshotVersion(long volumeId, String from, String to) {
-        TransactionLegacy txn = TransactionLegacy.currentTxn();
-        PreparedStatement pstmt = null;
-        String sql = UPDATE_SNAPSHOT_VERSION;
```
Comment (rafaelweingartner) [inline engine/schema/src/main/java/com/cloud/storage/dao/SnapshotDaoImpl.java]: The same happens here.

File: engine/schema/src/main/java/com/cloud/storage/dao/SnapshotDaoImpl.java [hunk 3]
```diff
@@ -253,41 +202,6 @@ public long getLastSnapshot(long volumeId, DataStoreRole role) {
...
-            s_logger.error("error getting last snapshot", ex);
-        }
-        return 0;
-    }
-
-    @Override
-    public long updateSnapshotSecHost(long dcId, long secHostId) {
-        TransactionLegacy txn = TransactionLegacy.currentTxn();
-        PreparedStatement pstmt = null;
-        String sql = UPDATE_SECHOST_ID;
```
Comment (rafaelweingartner) [inline engine/schema/src/main/java/com/cloud/storage/dao/SnapshotDaoImpl.java]: and here.

Comment (DaanHoogland) [reply engine/schema/src/main/java/com/cloud/storage/dao/SnapshotDaoImpl.java]: thanks, will do

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by borisstoyanov** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
A lot of unrelated errors here. I had a look at the snapshot testsuite and the exception has to do with creating a storage pool and not with the snapshotting itself. So also this failure is not related to this change.

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Pull request merged**